docs/references/config/config.toml.md

```diff
@@ -0,0 +1,1749 @@
+```
+
+| Value type          | string (comma-separated)         |
+|:--------------------|:---------------------------------|
+| **Possible values** | comma-separated list of node IDs |
+|                     | `""`                             |
+
+If a peer listed in this property requests a connection, it will be accepted, even if the
+[`p2p.max_num_inbound_peers`](#p2pmax_num_inbound_peers) or the
+[`p2p.max_num_outbound_peers`](#p2pmax_num_outbound_peers) are reached.
+
+Contrary to other settings, only the node ID has to be defined here, not the IP:port of the remote node.
+
+### p2p.persistent_peers_max_dial_period
+Maximum pause when redialing a persistent peer.
+```toml
+persistent_peers_max_dial_period = "0s"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0s"`      |
+
+If `"0s"` is set, then exponential backoff is applied to re-dial the remote node over and over.
+
+<!--- Todo: add if there are any limitations to the exponential back-off. --->
+
+### p2p.flush_throttle_timeout
+Time to wait before flushing messages out on the connection.
+```toml
+flush_throttle_timeout = "100ms"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0ms"`     |
+
+Write (flush) any buffered data to the connection. The flush is throttled, so if multiple triggers come in within the
+duration, only one flush is executed.
+
+Setting the value to `0ms` is possible, but the behaviour is undefined.
+
+<!--- Todo: trace the code and update what happens if this is set to 0. --->
+
+### p2p.max_packet_msg_payload_size
+Maximum size of a message packet payload, in bytes.
+```toml
+max_packet_msg_payload_size = 1024
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt; 0  |
+
+The value represents the maximum size of a message payload (or message data) in bytes.
+
+### p2p.send_rate
+Rate at which packets can be sent, in bytes/second.
+```toml
+send_rate = 5120000
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt; 0  |
+
+The value represents the amount of packet bytes that can be sent per second.
+
+### p2p.recv_rate
+Rate at which packets can be received, in bytes/second.
+```toml
+recv_rate = 5120000
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt; 0  |
+
+The value represents the amount of packet bytes that can be received per second.
+
+### p2p.pex
+Enable peer exchange reactor.
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `true`  |
+|                     | `false` |
+
+The peer exchange reactor is responsible for exchanging peer addresses among different nodes. If this is disabled, the
+node can only connect to addresses preconfigured in [`p2p.seeds`](#p2pseeds) or
```

Comment:
No, no seeds. Seeds are dialed by the PEX reactor: https://github.com/cometbft/cometbft/blob/main/p2p/pex/pex_reactor.go#L630